libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -785,7 +786,8 @@ size_t CodeTransform::variableHeightDiff(Scope::Variable const& _var, YulString
 			_varName.str() +
 			" is " +
 			to_string(heightDiff - limit) +
-			" slot(s) too deep inside the stack."
+			" slot(s) too deep inside the stack. " +
+			stackTooDeepString
```

Comment:
Can we have test case that covers this change?

Reply:
It's an internal compiler error, we don't usually test for them (see: https://github.com/ethereum/solidity/pull/12983#issuecomment-1125949942).

Reply:
Well, technically here it's just adding it to `m_stackErrors` which is exposed through `CodeTransform::stackErrors()` and that is sometimes used even when there's no ICE (mostly just to check if it's empty though).

I guess this change is simple enough that I'm fine not having a test here. I suspect that we do have some tests covering the mechanism itself and it's just that we don't have anything comparing the message specifically.